mllib/src/main/scala/org/apache/spark/mllib/feature/Word2Vec.scala

```diff
@@ -410,6 +410,9 @@ class Word2Vec extends Serializable with Logging {
       i += 1
     }
 
+    require(word2VecMap.size > 0, "The word2vec map should not be empty. You may need to check " +
```

Comment:
This should be checked by the `vocabSize`, but actually it doesn't. You are right, I think the check should be added in this line https://github.com/yinxusen/spark/blob/SPARK-7663/mllib/src/main/scala/org/apache/spark/mllib/feature/Word2Vec.scala#L160 to prevent the following computations.